load_balancers.go

```diff
@@ -138,6 +140,7 @@ type LoadBalancerRequest struct {
 	Algorithm              string           `json:"algorithm,omitempty"`
 	Region                 string           `json:"region,omitempty"`
 	SizeSlug               string           `json:"size,omitempty"`
```

Comment:
```
        // Deprecated: Use SizeUnit instead. 
	SizeSlug               string           `json:"size,omitempty"`
```

It would be great label `SizeSlug` as deprecated and include any relevant details so this shows up in godoc, etc.

Reply:
We will do this in the future. Currently we still expect SizeSlug to be used.